src/modm/architecture/interface/clock.hpp.in

```diff
@@ -97,13 +97,16 @@ using namespace ::std::chrono_literals;
 %% if with_io
 #include <modm/io/iostream.hpp>
 
+namespace modm {
```

Comment:
Formatting is not consistent here with the other ones where `{` is on the next line.